src/support/cleanse.cpp

```diff
@@ -32,7 +36,7 @@ void memory_cleanse(void *ptr, size_t len)
        might try to eliminate "superfluous" memsets. If there's an easy way to
        detect memset_s, it would be better to use that. */
 #if defined(_MSC_VER)
-    __asm;
```

Comment:
See the discussion here #11196 for context here. Now would be a good time to reassess.

Reply:
Note that __asm is not supported with msvc when targeting x64, that's the only reason for this change.